src/routes/newsRoute.js

```diff
@@ -5,14 +5,17 @@ const {testMsgFormat} = require('../webhook/testMsgFormat');
 const {setTestAddUserPostComment} = require('../webhook/setTestAddUserPostComment');
 const {setTestDeleteUserPostComment} = require('../webhook/setTestDeleteUserPostComment');
 const {setTestRegisterV2} = require('../webhook/setTestRegisterV2');
+const {setTestTopicAutoMessage} = require('../webhook/setTestTopicAutoMessage');
+
 const router = express.Router();
 
 router.post('/message-posted', createQueueNews);
-router.post('/test/message-format', testMsgFormat);
+// router.post('/test/message-format', testMsgFormat);
 // router.post("/test", serviceTestQueue);
 // router.post("/test-rss", testRss);
 // router.post("/test/credder", setTestCredder)
 // router.post("/test/add-user-post-comment", setTestAddUserPostComment)
 // router.post("/test/delete-user-post-comment", setTestDeleteUserPostComment)
-router.post('/test/register-v2', setTestRegisterV2);
+// router.post('/test/register-v2', setTestRegisterV2);
+// router.post('/test/topic-auto-message', setTestTopicAutoMessage);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `setTestTopicAutoMessage` is added correctly, but the corresponding route is commented out. If this is intentional and you plan to use it in the future, consider adding a comment explaining why it's commented out. If it's not needed, it would be better to remove both the import and the route to keep the code clean.
```diff
- const {setTestTopicAutoMessage} = require('../webhook/setTestTopicAutoMessage');
...
- // router.post('/test/topic-auto-message', setTestTopicAutoMessage);
```
If you intend to use it now, uncomment the route.
```diff
+ router.post('/test/topic-auto-message', setTestTopicAutoMessage);
```
Also, note that several other routes are commented out. If these routes are no longer needed, consider removing them to reduce clutter in your codebase.


<!-- This is an auto-generated comment by OSS CodeRabbit -->